using-marquee.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import time
+
+def main():
+    program = animate()
+    program.data = input("Enter string : ") + " "
```

Comment:
Change the data with a permanent string "Hello, World!" since the sole purpose of this repository is to print "Hello, World!"